config/default.js

```diff
@@ -29,6 +29,10 @@ module.exports = {
   site: {
     name: 'Punchcard CMS',
   },
+  login: {
+    loginPath: '/login',
```

Comment:
Can we work on naming things? Let's try and have as few camelCase names as we can. I'd prefer `users.paths.login` to `login.loginPath`